lib/app/register/register_controller.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:hnh/app/abstract/controller.dart';
+import 'package:hnh/domain/repositories/authentication_repository.dart';
+import 'package:hnh/app/register/register_presenter.dart';
+import 'package:hnh/domain/entities/user.dart';
+
+class RegisterController extends Controller {
+  String _firstName;
+  String _lastName;
+  String _userEmail;
+  String _userPassword;
+  dynamic _context;
+
+  AuthenticationRepository _authenticationRepository;
+  RegisterPresenter _registerPresenter;
+  User _currentUser;
+  User get currentUser => _currentUser;
```

Comment:
those are not being used

Reply:
the _registerPresenter is being used to call the OnComplete and OnError mothods? Right?

Reply:
The User has been removed.